planet/scripts/cli.py

```diff
@@ -103,3 +107,85 @@ def init(email, password):
     response = call_and_wrap(clientv1().login, email, password)
     write_planet_json({'key': response['api_key']})
     click.echo('initialized')
+
+
+@cli.command('quota')
+@click.option('-k', default=None, prompt=False, hide_input=True, help=(
+    'Enter your API Key'))
+def quota(k):
+    '''Print allocation and remaining quota in Sqkm.'''
+    try:
+        if k is not None:
+            main = requests.get(
+                'https://api.planet.com/auth/v1/' +
+                'experimental/public/my/subscriptions',
+                auth=HTTPBasicAuth(
+                    k, ''))
+            if main.status_code == 200:
+                content = main.json()
+                for item_id in content:
+                    click.echo(" ")
+                    click.echo(
+                        "Allocation Name: " + str(
```

Comment:
The more idiomatic way is using format strings. I personally prefer the `%s` variety but would reluctantly accept `{}`.

For example, `click.echo('Allocation Name: %s' % item_id['organization']['name'])`

Also: strings should be single quoted (as in example above)

Reply:
Made the changes Ian, thank you for the suggestions